aim/web/ui/src/components/MediaList/ImageBox.tsx

```diff
@@ -71,6 +71,7 @@ const ImageBox = ({
     e.stopPropagation();
     setIsImageFullViewPopupOpened(true);
   }
+
   // TODO need to add focused image logic
   // function safeSyncHoverState(args: any): void {
```

Comment:
Please remove comments.

Reply:
As we didn't add this functionality I will remove it, later if we need it i will add, thanks

Reply:
will remove, it's unnecessary for now , thanks